<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPassword</class>
 <widget class="QWidget" name="ForgetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>635</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QFrame{
	background: rgba(248, 248, 248, 220);
	border: 2px solid rgb(102, 102, 102);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica 12</family>
         <pointsize>15</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 2px solid rgb(15, 128, 255);
	background:rgba(15, 128, 255, 93);
	border-radius:10px;
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold;
}

QPushButton:hover{
	border: 2px solid rgb(15, 128, 255);
	background:rgba(15, 128, 255, 43);
	border-radius:10px;
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold;
}</string>
       </property>
       <property name="text">
        <string>Send Code</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica 12</family>
         <pointsize>15</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 2px solid rgb(179, 179, 179);
	border-radius:15px;
	background:rgba(244, 244, 244, 20);


	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold ;
}


QPushButton:hover{
	border: 2px solid rgb(179, 179, 179);
	border-radius:15px;
	background:rgb(240, 240, 240);
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold ;
}
</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>485</y>
      <width>321</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 2px solid rgb(200, 200, 200);
	border-radius:20px;
	background: rgba(248, 248, 248, 210);
	margin-top: 11 px;
}

QGroupBox::title {
	background: rgb(248, 248, 248);
	subcontrol-origin: margin;
	subcontrol-position: top center;	
	padding: 3 1px;
	color: rgba(15, 128, 255, 193);
	border: 2px solid rgba(15, 128, 255, 193);
	border-radius:10px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>28</y>
       <width>170</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border: 2px solid rgb(127, 127, 127);
	border-radius:10px;
	font: 75 16pt &quot;Helvetica&quot; bold;
	lineedit-password-mask-delay: 500;
}
</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica 12</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background: none;
	border: 0px;
	color: rgb(76, 76, 76);
	font: 75 16pt &quot;Helvetica&quot; bold;
}</string>
     </property>
     <property name="text">
      <string>New Pwd</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>78</y>
       <width>170</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border: 2px solid rgb(127, 127, 127);
	border-radius:10px;
	font: 75 16pt &quot;Helvetica&quot; bold;
	lineedit-password-mask-delay: 500;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>84</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica 12</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background: none;
	border: 0px;
	color: rgb(76, 76, 76);
	font: 75 16pt &quot;Helvetica&quot; bold;
}</string>
     </property>
     <property name="text">
      <string>Repeat</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>335</y>
      <width>320</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 2px solid rgb(200, 200, 200);
	border-radius:20px;
	background: rgba(248, 248, 248, 210);
	margin-top: 11 px;
}

QGroupBox::title {
	background: rgb(248, 248, 248);
	subcontrol-origin: margin;
	subcontrol-position: top center;	
	padding: 3 1px;
	color: rgba(15, 128, 255, 193);
	border: 2px solid rgba(15, 128, 255, 193);
	border-radius:10px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>84</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica 12</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background: none;
	border: 0px;
	color: rgb(76, 76, 76);
	font: 75 16pt &quot;Helvetica&quot; bold;
}</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica 12</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background: none;
	border: 0px;
	color: rgb(76, 76, 76);
	font: 75 16pt &quot;Helvetica&quot; bold;
}</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>78</y>
       <width>170</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border: 2px solid rgb(127, 127, 127);
	border-radius:10px;
	font: 75 14pt &quot;Helvetica&quot; bold;
}
</string>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>28</y>
       <width>170</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border: 2px solid rgb(127, 127, 127);
	border-radius:10px;
	font: 75 16pt &quot;Helvetica&quot; bold;
}
</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>495</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica 12</family>
      <pointsize>15</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background: none;
	border: 0px;
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold;
}</string>
    </property>
    <property name="text">
     <string>Security Code</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>567</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Helvetica 12</family>
      <pointsize>15</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 2px solid rgb(15, 128, 255);
	background:rgba(15, 128, 255, 93);
	border-radius:10px;
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold;
}

QPushButton:hover{
	border: 2px solid rgb(15, 128, 255);
	background:rgba(15, 128, 255, 43);
	border-radius:10px;
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold;
}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>518</y>
      <width>111</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(127, 127, 127);
	border-radius:10px;
background: rgba(248, 248, 248, 210);
	font: 75 16pt &quot;Helvetica&quot; bold;
}
</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>480</y>
      <width>507</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="titlepic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>324</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
